src/CodeCoverage.php

```diff
@@ -340,6 +340,14 @@ public function append(array $data, $id = null, $append = true, $linesToBeCovere
      */
     public function merge(PHP_CodeCoverage $that)
     {
+       $this->filter->setBlacklistedFiles(
```

Comment:
okay fixing that, did it quickly on github's editor, seemed a lot simpler than cloning the repo and doing all those things :D 
also sorry, didn't see ur second comment before I put mine